cmd/web.go

```diff
@@ -577,6 +583,26 @@ func runWeb(ctx *cli.Context) {
 		err = server.ListenAndServeTLS(setting.CertFile, setting.KeyFile)
 	case setting.FCGI:
 		err = fcgi.Serve(nil, m)
+	case setting.UNIX_SOCKET:
+		os.Remove(listenAddr)
+		listener, err := net.ListenUnix("unix", &net.UnixAddr{listenAddr, "unix"})
+		if err != nil {
+			break
+		}
+
+		// FIXME add proper implementation of signal capture on all protocols
+		sigc := make(chan os.Signal, 1)
```

Comment:
Sorry I did not say it clear, we do not catch SIGTERM right now, but leave a comment (you already have) to implement in the future.